### PR TITLE
Fix: handle broken symlinks

### PR DESCRIPTION
**Pull request by lili2311**
- [X] Ready for review
- [x] Follows [CONTRIBUTING](https://github.com/snyk/snyk/blob/master/.github/CONTRIBUTING.md) rules
- [x] Reviewed by Snyk internal team

#### What does this PR do?

Fixes an edge case when using "snyk test --all-projects" in a project that contains a broken symlink. (Currently it fails catastrophically.)

#### Where should the reviewer start?

Small change to src/lib/find-files.ts

#### How should this be manually tested?

$ ln -s /path-that-does-not-exist broken-symlink
$ node dist/cli/index.js test --all-projects

### CODE REVIEW

**Comment by clintonherget** [comment]
Superseded by https://github.com/snyk/snyk/pull/1335 which should allow existing tests to pass. :)

This one can be closed.

File: src/lib/find-files.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ import * as pathLib from 'path';
 import * as _ from '@snyk/lodash';
 import { detectPackageManagerFromFile } from './detect';
 import * as debugModule from 'debug';
-const debug = debugModule('snyk');
+const debug = debugModule('snyk:find-files');
```
Comment (orsagie) [inline src/lib/find-files.ts:6]: I really like this, we should do this for more debugs

**Review by orsagie — APPROVED** [review]
